data_steward/tools/generate_unioned_ehr_dataset.sh

```diff
@@ -68,29 +99,30 @@ source "${TOOLS_DIR}/set_path.sh"
 echo "-------------------------->Take a Snapshot of Unioned EHR Submissions (step 5)"
 source_prefix="unioned_ehr_"
 unioned_ehr_dataset="${dataset_release_tag}_unioned_ehr"
-echo "unioned_ehr_dataset --> $unioned_ehr_dataset"
+unioned_ehr_dataset_backup="${unioned_ehr_dataset}_backup"
+unioned_ehr_dataset_staging="${unioned_ehr_dataset}_staging"
 
 #---------------------------------------------------------------------
 # Step 1 Create an empty dataset
-bq mk --dataset --description "copy ehr_union from ${ehr_snapshot}" ${app_id}:${unioned_ehr_dataset}
+bq mk --dataset --description "copy ehr_union from ${ehr_snapshot}" --label "phase:backup" --label "release_tag:${dataset_release_tag}" --label "de_identified:false"${app_id}:${unioned_ehr_dataset_backup}
```

Comment:
add a space between `"de_identified:false"` and `${app_id}
```suggestion
bq mk --dataset --description "copy ehr_union from ${ehr_snapshot}" --label "phase:backup" --label "release_tag:${dataset_release_tag}" --label "de_identified:false" ${app_id}:${unioned_ehr_dataset_backup}
```